master/docs/manual/configuration/workers-libvirt.rst

```diff
@@ -180,7 +180,7 @@ Configure remote libvirt server:
        if vir_domain in domains:
            domain = domains[vir_domain]
            cmd = ['/usr/bin/qemu-img', 'create', '-b', images_path + domain[0],
-                  '-f', 'qcow2', images_path + domain[1]]
+                  '-f', 'qcow2', '-F', 'qcow2', IMAGES_PATH + domain[1]]
```

Comment:
Should be lowercase images_path, shouldn't it?

Reply:
Indeed, fixed!